Samples/CloudMirror/CloudMirror/FakeCloudProvider.cpp

```diff
@@ -104,6 +105,34 @@ void CALLBACK FakeCloudProvider::OnCancelFetchData(
     FileCopierWithProgress::CancelCopyFromServerToClient(callbackInfo, callbackParameters);
 }
 
+// copied from FileCopierWithProgress.cpp
+#define FIELD_SIZE( type, field ) ( sizeof( ( (type*)0 )->field ) )
+#define CF_SIZE_OF_OP_PARAM( field )                                           \
+    ( FIELD_OFFSET( CF_OPERATION_PARAMETERS, field ) +                         \
+      FIELD_SIZE( CF_OPERATION_PARAMETERS, field ) )
+
+void CALLBACK FakeCloudProvider::OnNotifyDelete(
+    _In_ CONST CF_CALLBACK_INFO* callbackInfo,
+    _In_ CONST CF_CALLBACK_PARAMETERS* callbackParameters)
+{
+    CF_OPERATION_INFO opInfo = { 0 };
+    CF_OPERATION_PARAMETERS opParams = { 0 };
+
+    wprintf(L"Acknowledging delete request %s\n", callbackInfo->NormalizedPath);
+
+    opInfo.StructSize = sizeof(opInfo);
+    opInfo.Type = CF_OPERATION_TYPE_ACK_DELETE;
+    opInfo.ConnectionKey = callbackInfo->ConnectionKey;
+    opInfo.TransferKey = callbackInfo->TransferKey;
+    opParams.ParamSize = CF_SIZE_OF_OP_PARAM(AckDelete);
+    opParams.AckDelete.CompletionStatus = 0;
+    opParams.AckDelete.Flags = CF_OPERATION_ACK_DELETE_FLAG_NONE;
+
+    // no issue with CfExecute during happy path.
```

Comment:
Thanks for implementing this callback in the sample. 
These comments aren't desirable in the sample though. Please remove.